samples/cpp/videocapture_audio_combination.cpp

```diff
@@ -0,0 +1,56 @@
+#include <opencv2/core.hpp>
+#include <opencv2/videoio.hpp>
+#include <opencv2/highgui.hpp>
+#include <iostream>
+#include <stdio.h>
+#include <thread>
+
+using namespace cv;
+using namespace std;
+
+int main(int, char**)
+{
+    Mat frame_video;
+    Mat frame_audio;
+    std::vector<Mat> video_data;
+    Mat audio_data;
+
+    int apiID = cv::CAP_MSMF;
+    //congigurate VideoCapture for video and audio
+    VideoCapture cap_video;
+    VideoCapture cap_audio;
+    // open selected camera using selected API
+    cap_video.open(0, apiID);
+    if (!cap_video.isOpened()) {
+        cerr << "ERROR! Can't to open camera\n";
+        return -1;
+    }
+    // open selected micro using selected API
+    std::vector<int> params { CAP_PROP_AUDIO_ENABLE , static_cast<int>(1) };
```

Comment:
> static_cast<int>(1)

why?

Reply:
I guided on creating a vector of parameters in a video writer (cap.cpp). There the bool cast to int, in my case it is not required. I will correct this.